transforms/SYSCTL_G350x_G310x_G150x_G110x.yaml

```diff
@@ -40,7 +40,16 @@ transforms:
 
   # TODO SYSTEMCFG: Flatten WWDTLP[0..1]RSTDIS
 
-  # TODO RSTCAUSE: Match other chips?
+  # RSTCAUSE.ID.SYSWWDT0 should actually be called BOOTWWDT0 and have a value of 14
```

Comment:
For consistency BOOTWWDT1?

Reply:
According to the tech reference, it's part of the SYSRST level:

<img width="671" height="227" alt="image" src="https://github.com/user-attachments/assets/91167101-764f-41e2-bd09-ccb5d5528c12" />